src/test/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcfTest.kt

```diff
@@ -277,6 +277,28 @@ class CreateFastaFromHvcfTest {
         }
     }
 
+    @Test
+    fun testCreateHaplotypeSequenceImputedVCf() {
```

Comment:
could change this to testCreateHaplotypeSequenceFromImputedVcf? When I read the method name the first time I thought it meant you were creating a VCF

Reply:
changed as requested.